lib/gauge_qcharge.cu

```diff
@@ -68,16 +70,16 @@ public:
 
         checkCudaError();
         comm_allreduce((double *)arg.result_h);
-        charge = arg.result_h[0];
+        charge = arg.result_h[0] / (-4*M_PI*M_PI);
       } else {
         QChargeArg<Float, nColor, recon, false> arg(Fmunu, (Float*)qDensity);
         QChargeCompute<decltype(arg)> qChargeCompute(arg, Fmunu);
         qChargeCompute.apply(0);
         qudaDeviceSynchronize();
-
+	
         checkCudaError();
         comm_allreduce((double *)arg.result_h);
-        charge = arg.result_h[0];
+        charge = arg.result_h[0] / (-4*M_PI*M_PI);
```

Comment:
@cpviolator I guess this is the source of the minus sign discrepancy with tmLQCD?

Reply:
And the minus sign agreement with MILC.